pkg/webui/console/store/actions/lib.js

```diff
@@ -55,8 +55,27 @@ export const attachPromise = actionOrActionCreator => {
     return decorateAction(actionOrActionCreator)
   }
 
-  return function(...args) {
+  return (...args) => {
     const action = actionOrActionCreator(...args)
     return decorateAction(action)
   }
 }
+
+/**
+ * Helper function to retrieve the result action types based
+ * on the request action type.
+ *
+ * @param {object|string} typeString - The request action type.
```

Comment:
Is it an array or an object?

Note that
```js
const obj = {};
obj instanceof Array // is `false`
```

```suggestion
 * @param {Array<...>|string} typeString - The request action type.
```